### PR TITLE
Readthrough1

### PR DESCRIPTION
**Pull request by jurecuhalev**

### CODE REVIEW

File: 1-installation/index.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ title: "Part 1: Let's get your computer ready!"
 permalink: /1-installation/
 ---
 
-You'll create your website with a framework called <a href="http://jekyllrb.com" target="_blank">Jekyll</a>. Despite the unusual name, Jekyll is a nice tool you can use to create simple, yet flexible websites that can grow with you as you learn more about various web technologies. 
+You'll create your website with a framework called <a href="http://jekyllrb.com" target="_blank">Jekyll</a>. Despite the unusual name, Jekyll is a nifty tool you can use to create simple, yet flexible websites that can grow with you as you learn more about various web technologies.
```
Comment (ialja) [inline 1-installation/index.md]: /me no likey nifty :stuck_out_tongue:

File: 3-pages/index.md [hunk 2]
```diff
@@ -44,7 +46,7 @@ The second part of `index.html` is more interesting. It's mainly written in HTML
 
 	</div>
 
-Looks confusing, but it's actually pretty simple. HTML is mainly used to define the structure of websites. Let's take a closer look at this line:
+Looks confusing, but it's simple if you follow the rules. HTML is mainly used to define the structure of websites. Let's take a closer look at this line:
```
Comment (ialja) [inline 3-pages/index.md]: Don't like the following rules part (too formal sounding imho).

File: 1-installation/index.md [hunk 3]
```diff
@@ -6,11 +6,11 @@ permalink: /1-installation/
 
 You'll create your website with a framework called <a href="http://jekyllrb.com" target="_blank">Jekyll</a>. Despite the unusual name, Jekyll is a nice tool you can use to create simple, yet flexible websites that can grow with you as you learn more about various web technologies. 
 
-Before we can get started, let's take a few minutes to get familiar with an app that allows you to talk to your computer directly and to install a few tools on your computer that will make your work easier.
+Before we can get started, let's take a few minutes to get familiar with an app that allows you to command your computer. We'll also install a few tools on your computer that will make your work easier.
 
 ## Step 1: Get to know the Terminal
 
-First, find and open an app called Terminal on your computer. It looks like something from a sci-fi movie, but don't be intimidated by its looks.
+First, find and open an app called Terminal on your computer. It's usually in `/Applications/Utilities`. It looks like something from a sci-fi movie, but don't be intimidated by its looks.
```
Comment (goranche) [inline 1-installation/index.md]: are we limiting ourselves to Macs? maybe instructions for Windows?
also, maybe instructions on how to open it using spotlight?

Comment (jurecuhalev) [inline 1-installation/index.md:13]: We are not. But first draft is written with Macs in mind, so I'm expanding this part. Windows version will reference Powershell anyway.

File: 1-installation/index.md [hunk 4]
```diff
@@ -136,7 +136,7 @@ To make sure we're all using the same tools, let's go ahead and install <a href=
 
 ## Step 5: Install GitHub
 
-This app will make it easier to publish your website online by the end of the day.
+This app will make it easier to publish your website online at the end of the day.
```
Comment (goranche) [inline 1-installation/index.md]: I'd leave "end of the day" out of it... you can deploy any time :wink:

File: 1-installation/index.md (hunk 3 above)
Comment (ialja) [inline 1-installation/index.md:13]: Windows instructions are on the to-do list, we'll also have to agree on a how to format them both if we want to have the ability to switch with a click (some simple JS probably). 

As to Spotlight - we have to find out how "regular" people find apps on their Macs ;) We'll test the process with various users, of course.

Comment (goranche) [inline 1-installation/index.md:13]: > we have to find out how "regular" people find apps on their Macs

meh, just teach them the "right" way :smile:

**Pull request merged**